### DISCUSSION TITLE
Restrict Request Size?

### DISCUSSION THREAD

**Discussion by thesec** [rhubarbgroup/.github#39] [Support (ROC)] [answered]
The max request size is 1 mb for me.

Is there already any option to split data into smaller chunks and send them in seperate commands?
How to do so?

**Comment by tillkruss** [comment]
What do you mean by `max request size` specifically?

**Reply by thesec** [reply]
Am just diving into redis ... :) 

Perhaps it's been a message your plugin returned from upstash which I am using for my testdrive.
Once the plugin was initiated, it reported an error, stating that the maximum request size was exceeded.

As per https://upstash.com/pricing they limit the size per "request / command" to 1 mb.

**Reply by tillkruss** [reply, marked as answer]
I see. Paying $$$ on a per-request basis is gonna get expensive.

Redis Object Cache does not chunk Redis commands, so if your request size is over 1MB, it's probably your `alloptions` key which is just a messy WordPress relic. I'd suggest using a different Redis host, or slimming down your WordPress app.

**Reply by thesec** [reply]
Thanks Till. 

Actually I am in parallel already climbing both trees ...

- looking for different host but struggling to identify one in Germany - any recommendations?
- and slimming down the blog installation - which are the usual suspects that'll help to reduce the bulk?

Seems like some work on 'alloptions' has already been started (https://jjj.blog/2017/04/understanding-alloptions-in-wordpress/) back then ... but did not make it into the core yet :/ 🤷🏻‍♂️

**Reply by tillkruss** [reply]
The guys at https://raidboxes.io are competent.
